src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx

```diff
@@ -369,19 +379,21 @@ const MinimumSize = ({ partition, units }: MinimumSizePropTypes) => {
     <ValidatedTextInput
       ariaLabel="minimum partition size"
       helperText=""
```

Comment:
Let's add some error message
![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/5f76ed0f-ff35-4e0f-b334-a0daa07d3305)


```suggestion
      helperText="Must be larger than 0"
```